ccs/interviews.md

```diff
@@ -0,0 +1,269 @@
+# Interviews API Endpoints
+
+* [Get Interview](#get-interviewsinterviewid)
+* [Get Interviews](#get-interviews)
+* [Create Interview](#post-interview)
+* [Update Interview](#put-interviewsinterviewid)
+* [Delete Interview](#delete-interviewsinterviewid)
+
+---
+
+### `GET` `/interviews/:id`
+* Returns an interview from the CCS.
+
+####  Header
+key|value
+---|---
+`Authorization`| Token
```

Comment:
The docs currently say
```
`Authorization`| Basic authorization (`Basic ` + base64(`api key` + `:`))
```
in some places and
```
`Authorization`| Token
```
in other places.

Is there a reason we're using two different formats?

Reply:
good catch. Thank you, Subrat